diff --git a/package/libaltsound/libaltsound.mk b/package/libaltsound/libaltsound.mk
index 9c84fb7..35287f2 100644
--- a/package/libaltsound/libaltsound.mk
+++ b/package/libaltsound/libaltsound.mk
@@ -10,7 +10,7 @@ LIBALTSOUND_LICENSE = BSD-3-Clause
 LIBALTSOUND_LICENSE_FILES = LICENSE
 LIBALTSOUND_DEPENDENCIES =
 LIBALTSOUND_SUPPORTS_IN_SOURCE_BUILD = NO
-# LIBALTSOUND_EXTRA_DOWNLOADS = https://www.un4seen.com/files/bass24-linux.zip
+LIBALTSOUND_EXTRA_DOWNLOADS = https://www.un4seen.com/files/bass24-linux.zip
 
 LIBALTSOUND_CONF_OPTS += -DCMAKE_BUILD_TYPE=Release
 LIBALTSOUND_CONF_OPTS += -DBUILD_STATIC=OFF
